nvm.sh

```diff
@@ -2239,8 +2239,6 @@ nvm_get_artifact_compression() {
   COMPRESSION='tar.gz'
   if [ "_${NVM_OS}" = '_win' ]; then
     COMPRESSION='zip'
-  elif nvm_supports_xz "${VERSION}"; then
-    COMPRESSION='tar.xz'
```

Comment:
it defaults to tar dot G z, and these lines set it to tar dot X z.

It's an entirely different format, and is quite intentional so the smaller xz format can be downloaded instead.